lib/src/main/java/com/faskn/lib/ClickablePieChart.kt

```diff
@@ -239,6 +257,8 @@ class ClickablePieChart @JvmOverloads constructor(
         val popupWindowY =
             (currentViewLocation[1] + halfRadius.toInt()) + calculatedY -
                     (if (calculatedY < 0) -halfOfSliceWidth else halfOfSliceWidth)
+
+        popupWindow.setBackgroundDrawable(ColorDrawable())
```

Comment:
Why we need this?

Reply:
The issue is after popup got showed I was unable to dismiss popup when I clicked outside of popup on API 21. Then I found this [fix](https://stackoverflow.com/a/12232775/6771753).